src/main/rust/bridge/meta_preview.rs

```diff
@@ -86,6 +89,11 @@ impl ToProtobuf<types::MetaItemPreview, (&Ctx, &ResourceRequest)> for MetaItemPr
                 .get(&self.id)
                 .map(|library_item| !library_item.removed)
                 .unwrap_or_default(),
+            in_cinema: self
```

Comment:
Logic is based on https://github.com/Stremio/stremio-models/blob/9b24ff5e05dd03f8c8b198bd0e4b4a25bbacfd12/metadata.js#L381-L389